ynput/ayon-core
# Publisher: Revert to default value

## Changelog Description
Implemented context menu option on attribute definitions to reset attribute values to default states.

## Additional info
All attribute definition widgets should have implemented logic to be able to reset value of attributes to default value. In Publisher it does reset to default value per instance.

## Testing notes:
1. Open Publisher in host of your choice.
2. Create instance/s.
3. In Publish tab select instance/s and change their values.
4. When you right click them and choose `Revert to default` it should set value to default value defined by create/publish plugin. It should also work on multiselection.
